cfme/infrastructure/config_management.py

```diff
@@ -264,7 +269,8 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
         if not cancel:
             flash_msg = version.pick({
                 version.LOWEST: 'Delete initiated for 1 Provider from the CFME Database',
-                '5.6': 'Delete initiated for 1 provider'
+                '5.6': 'Delete initiated for 1 provider',
+                '5.7': 'Delete initiated for 1 Provider'
```

Comment:
Leave only the `5.6` and `5.7` options in (change the `5.6` into `version.LOWEST`). The last one is not needed anymore.